config.yaml

```diff
@@ -288,7 +288,16 @@ options:
       profile will be extracted from '/var/log/cloud-init-output.log', fallback is 'level1_server'.
       Options are: '' (disable profile check), 'level1_server', 'level2_server',
                    'level1_workstation' or 'level2_workstation'
+      This option cannot be used with cis_audit_tailoringfile at the same time.
```

Comment:
What happens if both options are set?  It would be nice to document it here.  (eg. does the charm go into blocked state? does one option override the other?)

Reply:
I've changed the code to be into blocked state. Thanks for the suggestion